src/Build/Definition/ProjectCollection.cs

```diff
@@ -434,7 +434,7 @@ public static ProjectCollection GlobalProjectCollection
         /// This is the Windows file version (specifically the value of the FileVersion
         /// resource), not necessarily the assembly version.
         /// If you want the assembly version, use Constants.AssemblyVersion.
-        /// This is not the <see cref="P:Microsoft.Build.BuildEngine.ToolsetCollection.ToolsVersions*">ToolsetCollection.ToolsVersions</see>.
+        /// This is not a <see cref="Toolset"/>.
```

Comment:
I think this is the modern equivalent:

```suggestion
        /// This is not the <see cref="Toolset.ToolsVersion"/>.
```